src/FiltrationGroupComponent/FiltrationGroupComponent.tsx

```diff
@@ -181,8 +177,10 @@ export class FiltrationGroupComponent extends React.Component<Props, State> {
 		rootElement: HTMLElement,
 		searchClasses: string[]
 	) => {
+		const children = Array.from(rootElement.childNodes);
 		const items: ItemsRootElement[] = [];
-		rootElement.childNodes.forEach((item: HTMLElement) => {
+
+		children.map((item: HTMLElement) => {
```

Comment:
Тут лучше не использовать лишний `items.push`, а сразу возвращать отфильтрованный массив
```suggestion
		return children.filter((item: HTMLElement) => {
```